README-install.md

```diff
@@ -119,16 +119,16 @@ openshift-ansible-callback-plugins openshift-ansible-filter-plugins
 openshift-ansible-lookup-plugins openshift-ansible-playbooks
 openshift-ansible-roles
 
-    # yum install openshift-ansible \
-      openshift-ansible-callback-plugins openshift-ansible-filter-plugins \
-      openshift-ansible-lookup-plugins openshift-ansible-playbooks \
-      openshift-ansible-roles
+        # yum install openshift-ansible \
+          openshift-ansible-callback-plugins openshift-ansible-filter-plugins \
+          openshift-ansible-lookup-plugins openshift-ansible-playbooks \
+          openshift-ansible-roles
 
 If the 3.6 version of these packages are not available, you can use the
 git repo `https://github.com/openshift/openshift-ansible.git` and the
 `release-3.6` branch:
 
-    # git clone https://github.com/openshift/openshift-ansible.git -b release-3.6
+        # git clone https://github.com/openshift/openshift-ansible.git -b release-3.6
```

Comment:
these look ok now, and do not appear to need extra indentation: https://github.com/ViaQ/Main/blob/master/README-install.md#installing-viaq-packages